### DISCUSSION TITLE
Crash with Xcode 14.1

### DISCUSSION THREAD

**Discussion by mackoj** [pointfreeco/swift-url-routing#60] [Q&A] [answered]
Hi,

I see that you have a fix for Xcode 14.1 (https://github.com/pointfreeco/swift-url-routing/commit/80e8a0257ccdd639e31f709954ceca6b690fdc67).
Could you do a release 🙏 ?

Thanks,

**Comment by kamcma** [comment, marked as answer]
I think the linked fix is included in and specifically mentioned in the release notes of 0.3.1.

**Reply by mackoj** [reply]
My SPM didn't pull it 😂 thanks.
